src/Core/Settings.cs

```diff
@@ -313,10 +313,13 @@ public class ThemesImpl : SettingsOptionsAttribute.AbstractImpl
         public bool AutoSwapImagesIncludesFullView = false;
 
         [ConfigComment("A list of what buttons to include directly under images in the main prompt area of the Generate tab.\nOther buttons will be moved into the 'More' dropdown.\nThis should be a comma separated list."
-            + "\nThe following options are available: \"Use As Init\", \"Edit Image\", \"Upscale 2x\", \"Star\", \"Reuse Parameters\", \"Open In Folder\", \"Delete\", \"View In History\""
+            + "\nThe following options are available: \"Use As Init\", \"Edit Image\", \"Upscale 2x\", \"Star\", \"Reuse Parameters\", \"Open In Folder\", \"Delete\", \"View In History\", \"Refine Image\""
             + "\nThe default is blank, which currently implies 'Use As Init,Edit Image,Star,Reuse Parameters'")]
         public string ButtonsUnderMainImages = "";
 
+        [ConfigComment("The amount of image creativity used by the Upscale 2x button.")]
+        public double UpscaleCreativity = 0.4;
```

Comment:
This doesn't particularly make sense to have - Init Image Creativity is the value that gets used, the one in the JS is just a default that's lower than the standard default